LifeCycleAssessment_Engine/Compute/LCAToolPart.cs

```diff
@@ -0,0 +1,891 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using BH.oM.Structure.MaterialFragments;
+using System.IO;
+using System.Text;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Compute
+    {
+        [Description("The LCA_tool_part_tostart.py script in c#.")]
+        [Input("filePath", ".")]
+        [Input("c2_input", ".")]
+        [Input("d2_input", ".")]
+        [Input("b2_input", ".")]
+        [Input("e2_input", ".")]
+        [Input("g2_input", ".")]
+        [Input("h2_input", ".")]
+        [Input("f2_input", ".")]
+        [Input("o2_input", ".")]
+        [Input("i2_input", ".")]
+        [Input("j2_input", ".")]
+        [Input("m2_input", ".")]
+        [Input("l2_input", ".")]
+        [Input("w2_input", ".")]
+        [Input("p2_input", ".")]
+        [Input("u2_input", ".")]
+        [Input("t2_input", ".")]
+        [Input("q2_input", ".")]
+        [Input("n2_input", ".")]
+        [Input("s2_input", ".")]
+        [Input("v2_input", ".")]
+        [Input("c5_2ndlayer", ".")]
+        [Input("c6_2ndlayer", ".")]
+        [Input("c18_2ndlayer", ".")]
+        [Output("Success", "has it written to the file?.")]
```

Comment:
I don't like to laugh @EKAdebo but I feel I have to given the backstory on these full stops - but it looks like you have one too many on this occasion 🤣